### ISSUE TITLE
App bar buttons do nothing when viewed from a web browser

### ISSUE THREAD

**Issue by Harjot1Singh** [ShabadOS/core#22]
The minimise, fullscreen, and close buttons do nothing when accessing the display page from a web browser. Perhaps it's better to just not show the bar at all in this case?

**Comment by bhajneet** [comment]
It is most probably better to hide it in the web browser. However, I really don't think that many people have a use case aside from having to manually connect via phone's web browser. 

Since we would rather focus on creating an app, for an easier way to connect via GUI, this would be a very low priority. Please let me know if I'm mistaken, thank you.

**Comment by Harjot1Singh** [comment]
This is definitely a low priority, however, I reckon it would be better to keep the focus of the entire project as a fully-functional responsive web-app as you have now, and simply provide the app as an optional wrapper. 

On a side note about an app, I think it's also important to think about how exposing a "connect to" over the network may open up abuse/messing about from anyone not so sensible in the sangat.

Regarding the connect to feature on a desktop, see my comment https://github.com/ShabadOS/core/issues/7#issuecomment-354185943.

**Comment by bhajneet** [comment]
Fully functional web-app is nice, and I think definitely a goal of this project. At the same time, I still would like to reason that non-techies are more comfortable with using an app. Having a fully functional web-app will allow us to run go routines online so that people can use Shabad OS with just an internet connection. A cool idea for the future, and something we would like to work on later, so I think that should re-assure you on our commitment to being mostly a web-app.

Again, I agree that the web-app could use more cleaning up and a little more love. Just currently given the few users we have I really think we could benefit them with other work first.

Connect to is dangerous for sangat trolls. I would very much like to believe there are none, but in whatever chance that it is possible, we will implement a PIN connect to feature which the sevadaar can share with others personally.

**Comment by Harjot1Singh** [comment]
Of course, agree with everything you’ve stated. 

Perhaps there’s also a marketing element involved - STTM has a larger market share purely because people know of the name already. 

Is the issue list you have on GitHub complete, or is there other priority work that needs to be carried out?

**Comment by bhajneet** [comment]
The issue list is not complete, but I can work on adding everything we've talked about internally. It still would not be a perfect list as we are constantly in flux with ideas and roadmaps.

Marketing is a huge element. Most people will upgrade to STTM 3.0 simply because of the name. However, we are working on some ideas to start growing our userbase. We would like to do it mostly organically until we are sure we have a few things in place. 

I would say we need a little more testing on our macOS app. We have an installer we've shared internally and even have the code-signing license etc, but just haven't gotten around to doing that for electron-builder.

Another thing I would like before we start heavily marketing is the documentation which you suggested and social media "tutorials" that show off features and short "how-to's". This would provide a solid support backbone for new users.

**Comment by bhajneet** [comment]
I apologize there are some glaring other topics which need to be done. Namely, decoupling the database and settings.json so that they are not double-downloaded for every update even though they are the same file. In the db case it is just a lot of wasted bandwidth and time. For the settings it is just an unnecessary "reset" of options that people may have customized.
